source/_posts/2019-11-10-hexo-generator-feed-2-1_hexo-util-1-5.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Official plugins hexo-generator-feed 2.1.0 & hexo-util 1.5.0 released
+---
+
+We have released a new version of the official plugins [hexo-generator-feed] & [hexo-util].
+
+[hexo-generator-feed]: https://github.com/hexojs/hexo-generator-feed
+[hexo-util]: https://github.com/hexojs/hexo-util
+
+## hexo-generator-feed 2.1.0
+
+### Breaking change
+- Requires Node 8.10 or newer [#99]
```

Comment:
I might add a troubleshooting guide about Node.js version (`The engine "node" is incompatible with this module. Expected version`) in a separate PR.